### DISCUSSION TITLE
The page doesnt recognize my school from de image

### DISCUSSION THREAD

**Discussion by nesoto** [community/community#69639] [GitHub Education]
I dont know what to do, the image contains everything that i need to apply but the sw doesnt recognize anything so i idk what to do

**Comment by santoshvandari** [comment]
Hi "nesoto,"

I understand that you're having trouble with the GitHub Education verification process not recognizing the content in the image of your academic documentation. Here are some steps you can take to address this issue:

Image Quality: Ensure that your uploaded image is of high quality and clarity. Ensure that the text on the document is legible and that there are no obstructions or reflections that might interfere with the recognition process.

Supported Formats: Check if the image you've uploaded is in a supported format (e.g., JPEG, PNG, PDF). GitHub Education typically accepts standard image formats, so make sure your document is in one of these formats.

Content Visibility: Verify that all the necessary information, such as your name, school name, and other relevant details, is visible in the image. Sometimes, cropping or scaling the image incorrectly can lead to recognition issues.

Try a Different Image: If you believe the image you uploaded is explicit and contains the required information but is still not being recognized, you might want to try taking a new photo or using a different image. Ensure that the new image follows the guidelines mentioned above.

Contact GitHub Support: If none of the above steps resolve the issue and you continue to experience difficulties, it's advisable to contact GitHub Support for assistance. They can provide specific guidance or manually review your verification documents if necessary. You can reach out to GitHub Support here: [GitHub Education Support](https://support.github.com/contact/education).

Patience: Verification processes can sometimes take time. Be patient while GitHub reviews your documentation, and check your email for updates or instructions regarding your application.

Remember that GitHub Education is committed to helping students access their benefits, so reaching out to their support team can be pretty helpful in resolving verification issues.

I hope these suggestions help you in resolving the problem. If you have any more questions or need further assistance, please feel free to ask.
